### PR TITLE
Fix nightly tests

### PR DESCRIPTION
**Pull request by vanwagonet**
This fixes a missing event bug in planner & todo going to speedgrader

[run-nightly]
refs: MBL-14777
affects: none
release note: none

### CODE REVIEW

File: rn/Teacher/src/modules/to-do/list/ToDoList.js [hunk 1]
```diff
@@ -147,7 +147,7 @@ export class ToDoList extends Component<Props, State> {
     if (item.quiz) {
       assignmentID = item.quiz.assignment_id
     }
-    const path = `/courses/${item.courseID}/assignments/${assignmentID}/submissions`
+    const path = `/courses/${item.courseID}/assignments/${assignmentID}/submissions/speedgrader`
```
Comment (armstrongnate) [inline rn/Teacher/src/modules/to-do/list/ToDoList.js:150]: Thanks for fixing this

**Comment by inst-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  This pull request will not generate a release note.
  </td>
    </tr>
  </tbody>
</table>


Coverage | New % | Master % | Delta
-------- | ----- | -------- | -----
**Canvas iOS** | 93.23% | 93.23% | 0%

[MBL-14777](https://instructure.atlassian.net/browse/MBL-14777)

#### Affected Apps: None
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Comment by inst-danger** [comment]
<details><summary>Student.ipa</summary>
  <a href="https://app.bitrise.io/artifact/52065328/p/179a76e917052cd7319bf3944446d73a">
    
  </a>
</details>
<details><summary>Teacher.ipa</summary>
  <a href="https://app.bitrise.io/artifact/52065330/p/55dea19c3e299fc3cc91c15111682fb5">
    
  </a>
</details>
<details><summary>Parent.ipa</summary>
  <a href="https://app.bitrise.io/artifact/52065321/p/e2c592647a7e228e2abb2c5291ae3fa7">
    
  </a>
</details>

**Comment by vanwagonet** [comment]
⚠️ All of the test failures on bitrise are succeeding for me locally. 🤦‍♂️

**Review by armstrongnate — APPROVED** [review]

**Pull request merged**